.github/workflows/dvc-diff.yml

```
@@ -0,0 +1,53 @@
+# Checks for image diffs in a Pull Request and adds a GitHub comment showing the diff
+name: DVC image diff
+
+on:
+  pull_request:
+    paths:
+      - 'pygmt/tests/baseline/*.png.dvc'
+
+jobs:
+  dvc-diff:
+    name: DVC image diff
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2.3.4
+      with:
+        # fetch all history so that git diff works
+        fetch-depth: 0
+
+    - name: Setup data version control (DVC)
+      uses: iterative/setup-dvc@v1
```

I think dependbot doesn't like versions like `v1`.
```suggestion
      uses: iterative/setup-dvc@v1.0.2
```